### ISSUE TITLE
attempting to color a Relay in-world causes a freeze

### ISSUE THREAD

**Issue by boneskull** [refinedmods/refinedstorage#2770]
#### Issue description:

(Note that the relay in question was a couple blocks away from a disk manipulator, which has a known texture issue)

When right-clicking on a Relay with a Green Dye, Minecraft freezes and I have to force-quit

#### What happens:

Relevant exception:

```
[21:47:21] [Render thread/FATAL]: Preparing crash report with UUID 08827b49-07d1-4401-8bd0-9c6fe67fb3fe
[21:47:21] [Render thread/FATAL]: Unreported exception thrown!
java.lang.IllegalArgumentException: Cannot get property DirectionProperty{name=direction, clazz=class net.minecraft.util.Direction, values=[]} as it does not exist in Block{refinedstorage:relay}
	at net.minecraft.state.StateHolder.func_177229_b(SourceFile:95) ~[?:?]
	at com.refinedmods.refinedstorage.util.ColorMap.getNewState(ColorMap.java:116) ~[?:1.9.9]
	at com.refinedmods.refinedstorage.util.ColorMap.changeBlockColor(ColorMap.java:110) ~[?:1.9.9]
	at com.refinedmods.refinedstorage.block.RelayBlock.func_225533_a_(RelayBlock.java:38) ~[?:1.9.9]
	at net.minecraft.block.AbstractBlock$AbstractBlockState.func_227031_a_(AbstractBlock.java:638) ~[?:?]
	at net.minecraft.client.multiplayer.PlayerController.func_217292_a(PlayerController.java:288) ~[?:?]
	at net.minecraft.client.Minecraft.func_147121_ag(Minecraft.java:1366) ~[?:?]
	at net.minecraft.client.Minecraft.func_184117_aA(Minecraft.java:1645) ~[?:?]
	at net.minecraft.client.Minecraft.func_71407_l(Minecraft.java:1460) ~[?:?]
	at net.minecraft.client.Minecraft.func_195542_b(Minecraft.java:953) ~[?:?]
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:607) [?:?]
	at net.minecraft.client.main.Main.main(Main.java:184) [?:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_51]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_51]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_51]
	at java.lang.reflect.Method.invoke(Method.java:497) ~[?:1.8.0_51]
	at net.minecraftforge.fml.loading.FMLClientLaunchProvider.lambda$launchService$0(FMLClientLaunchProvider.java:51) [forge-1.16.4-35.1.4.jar:35.1]
	at net.minecraftforge.fml.loading.FMLClientLaunchProvider$$Lambda$488/883636619.call(Unknown Source) [forge-1.16.4-35.1.4.jar:35.1]
	at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:37) [modlauncher-8.0.6.jar:?]
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:54) [modlauncher-8.0.6.jar:?]
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:72) [modlauncher-8.0.6.jar:?]
	at cpw.mods.modlauncher.Launcher.run(Launcher.java:82) [modlauncher-8.0.6.jar:?]
	at cpw.mods.modlauncher.Launcher.main(Launcher.java:66) [modlauncher-8.0.6.jar:?]
```

#### What you expected to happen:

The Relay becomes a Green Relay

#### Steps to reproduce:
1. Place a Relay in the world
2. Right-click on it with a Green Dye
3. Freeze
...

Unsure if this happens with other dyes, but I can reproduce it consistently.

#### Version (make sure you are on the latest version before reporting):

- Minecraft: v1.16.4
- Forge: 35.1.4
- Refined Storage: 1.9.9

Does this issue occur on a server? [yes/no] no

**Comment by lauash** [comment]
Can confirm this also has affected me when attempting to dye a network transmitter and receiver that were placed in the world. The crash report is near identical to the one above. Just wanted to let you know it isn't just the relay, I've successfully dyed the wireless transmitter, and the disk manipulator.

**Comment by raoulvdberge** [comment]
Fixed for v1.9.10
